content/03-starting-oss-projects.md

```diff
@@ -143,11 +143,19 @@ You should have a policy for which of these way you use when. "Inbound=Outbound"
 
 ### Project governance
 
-See [issue #12](https://github.com/Open-Source-Compliance/outbound-oss/issues/12)
+An important factor for the success of an open source project is its governance. That comprises the rules, policies, conventions, and culture governing the collaboration. It determines factors such as how decisions are taken, who is in control, or who can join a project.
 
-open governance, transparency about governance
+In existing projects governance often has emerged over time, has gone from informal procedures driven by the practices of the project founders to more formally defined governance documented in contribution guides or ultimately instituted through a foundation as formal organization hosting the project.
 
-Company-based vs. foundation-driven, when to select which approach, leveraging a foundation (such as Eclipse)
+When starting a new open source project you can decide about how its governance will look like. This goes beyond deciding on a license. You will also have to decide about ownership of assets such as trademarks or domains and the rules how they can be used. And you will have to decide about policies of how people can become committers or maintainers, how releases and roadmaps are made, or how transparent the decision making process is.
```

Comment:
Better: "... you must decide ..." or "... you should decide ..." instead of "... you can decide ..."

Reply:
Changed it.